dev/release/post-04-website.sh

```diff
@@ -29,6 +29,12 @@ if [ "$#" -ne 2 ]; then
   exit 1
 fi
 
+# Verify archery is available and stop if not
+if [ -z "$(which archery)" ]; then
```

Comment:
How about executing `archery` instead of checking path?

```suggestion
if archery --help > /dev/null 2>&1; then
```

Reply:
Accepted. Just in the spirit of learning, what are the advantages to your version?

Reply:
My version can detect `archery` exists in `$PATH` but it doesn't work. (For example, if `python3` that is used for `pip install -e dev/archery` is uninstalled, `archery` exists but it doesn't work.)

Reply:
Ah, that's a good reason. Thanks.